### ISSUE TITLE
Types Error

### ISSUE THREAD

**Issue by Pedromdsn** [MathiasGilson/Tailwind-Styled-Component#75]
Good afternoon,
I'm having a problem with the lib typing.
I create a div, but it is defined as `any`, would it be possible to force the typing to div?

![image](https://user-images.githubusercontent.com/57453260/184507591-60dd70af-7054-4a3c-a7bf-ef4f0e8fbe6a.png)
(Don't care about styles , it's merely indicative xD)

**Comment by max-sym** [comment]
Yes, we're still waiting for the library to get updated on this issue.
BTW it's a duplicate of https://github.com/MathiasGilson/Tailwind-Styled-Component/issues/55
